src/listpack.c

```diff
@@ -787,25 +885,79 @@ unsigned char *lpInsert(unsigned char *lp, unsigned char *ele, uint32_t size, un
     return lp;
 }
 
+/* This is just a wrapper for lpInsert() to insert a string, which is
+ * Possible a integer encoded strings. */
```

Comment:
maybe mention that it can used to replace and delete too, and refer to lpInsert for details.

Reply:
`lpInsertString` has been removed.